docs/agents/sematext-agent/releasenotes.md

```diff
@@ -29,6 +33,30 @@ NULL
 
 -->
 
+# Version 2.1.0
+
+## New Features
+
+- OpenSearch logs parsing support: With this update, we now parse fields of your log messages. We will also be releasing OpenSearch Logpack soon to enhance your logging experience.
+- Proactive Log Monitoring functionality: Our product now includes a robust logging feature that enables our team to capture customer logs and proactively monitor their environment for potential issues.
+
+## Improvements
+
+- Additional Kubelet metrics: You can now collect a wider range of metrics from your Kubelet nodes, providing you with greater visibility into the health and performance of your Kubernetes cluster.
+- Logging process improvements: We have made several enhancements to our logging process to ensure smoother and more efficient logging for the Fleet.
+- Updated Vector version to 0.23 for Docker and Linux (baremetal) environments: This update ensures better compatibility with your environment and improved performance.
+
+## Bug Fixes
+
+- Fixed missing Docker container memory issue: We have resolved the issue where the Docker container memory was not being properly displayed.
```

Comment:
It would be good to add SC-XXXX here. Even if it's internal it can help us follow up or refer to a bug fix in the future.

Reply:
to ensure smoother and more efficient logging for the Fleet. ==> to reduce noise and simplify logs shown in Fleet.